### DISCUSSION TITLE
Convert pdf with watermark to tiff

### DISCUSSION THREAD

**Discussion by bhupendra-chetu** [ImageMagick/ImageMagick#2274] [Help]
Hi, I'm able to convert pdf to tiff using c#, but in the case of watermarked pdf, image becomes black except watermark. Any solution? Thanks in advance!

**Comment by dlemstra** [comment]
Can you share your input image and demonstrate the issue in more detail?

**Reply by bhupendra-chetu** [reply]
I used below code. Attached pdf has watermark.

public string ConvertToTiff(HttpPostedFileBase file, string filePath)
        {
            var fName = file.FileName.Substring(0, file.FileName.LastIndexOf('.'));
            using (MagickImageCollection images = new MagickImageCollection())
            {
                MagickReadSettings settings = new MagickReadSettings()
                {
                    Density = new Density(300),
                    Format = MagickFormat.Pdf
                };
                settings.Format= MagickFormat.Pdf;
                images.Read(file.InputStream, settings);
                int count = images.Count;
                for (int i = 0; i < count; i++)
                {                   
                    images.Write(filePath + "" + fName + ".tif");
                }
            }           
            var path = filePath + fName + ".tif";           
            return path;
        }

[Watermarked Pdf.pdf](https://github.com/ImageMagick/ImageMagick/files/4926538/Watermarked.Pdf.pdf)

After converting into tiff file, all page area is black except watermark.

**Reply by dlemstra** [reply]
What are you using the view the image? If you are using the Windows image viewer or paint it is possible that the transparent background is drawn as black. And when you convert the image to a JPEG file something similar will happen.
